src/header/header.scss

```diff
@@ -172,7 +173,8 @@
   @at-root {
     a#{&}, // rendered as anchor element
     button#{&}, // rendered as button element
-    &[role='button'] { // rendered as div with role=button
+    &[role='button'] {
+      // rendered as div with role=button
```

Comment:
I'd say remove this comment as it does not make sense on the next line, and prettier won't allow it on the same line